### DISCUSSION TITLE
Canr run the project on M1 mac

### DISCUSSION THREAD

**Discussion by SaabirMohamed** [Imperial-lord/mapbox-flutter#8] [Q&A]
Hi can you please assist me ..im scaffolding a project using the "get create project" ... as soon as i add the mapbox_gl and the mapbox navigation packages the app will never run on the simulator 
with the following error:
even if i clone the repo put in the keys ....still same issue 1st its the IOS DEPLOYMENT TARGET which i set to 11 ..but then the below error
Swift Compiler Error (Xcode): 'Flutter/Flutter.h' file not found
/Users/saabir/Documents/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_mapbox_navigation-0.0.26/ios/Classes/FlutterMapboxNavigationPlugin.h:0
:8

Swift Compiler Error (Xcode): Could not build Objective-C module 'flutter_mapbox_navigation'

Could not build the application for the simulator.
Error launching application on iPhone 12 Pro.

any help in this regard will be awesome thanks

**Comment by Imperial-lord** [comment]
Hey!

The plugin `flutter_mapbox_navigation` is not yet optimised for running on M1 Macs - I think I had mentioned this in the video itself. You can try running it on a real iOS device, or in an emulator. A M1 simulator will give you the same error. It however works on Intel Macs.
